torch/csrc/cuda/shared/cudart.cpp

```diff
@@ -30,7 +30,7 @@ void initCudartBindings(PyObject* module) {
   // By splitting the names of these objects into two literals we prevent the
   // HIP rewrite rules from changing these names when building with HIP.
 
-#if !defined(USE_ROCM)
+#if !defined(USE_ROCM) && CUDA_VERSION < 12000
```

Comment:
and this would prevent profiler initialization?

Reply:
Thanks! Just add some fix.